modules/wireguard-monitoring/default.nix

```diff
@@ -80,10 +80,23 @@ in {
               replacement = "$2";
             }
           ];
-        }];
+        }] ++ (if config.services.nginx.enable
+            then [{
```

Comment:
The formatting feels a bit unnatural for me, but I cannot come up with a better alternative, so feel free to ignore my concern

Reply:
`nixpkgs-fmt` suggests
```
diff --git a/modules/wireguard-monitoring/default.nix b/modules/wireguard-monitoring/default.nix
index 8f48559..cd40f74 100644
--- a/modules/wireguard-monitoring/default.nix
+++ b/modules/wireguard-monitoring/default.nix
@@ -3,7 +3,8 @@
 let
   wireguard-ip = config.wireguard-ip-address;
   common = import ./common.nix args;
-in {
+in
+{
   inherit (common) options;
 
   config = {
@@ -81,17 +82,17 @@ in {
             }
           ];
         }] ++ (if config.services.nginx.enable
-            then [{
-              job_name = "nginx-error-logs";
-              static_configs = [{
-                targets = [ "localhost" ];
-                labels = {
-                  job = "nginx-error-logs";
-                  host = config.networking.hostName;
-                  __path__ = "/var/log/nginx/*error.log";
-                };
-              }];
-            }] else []);
+        then [{
+          job_name = "nginx-error-logs";
+          static_configs = [{
+            targets = [ "localhost" ];
+            labels = {
+              job = "nginx-error-logs";
+              host = config.networking.hostName;
+              __path__ = "/var/log/nginx/*error.log";
+            };
+          }];
+        }] else [ ]);
       };
     };
```
Which probably looks a bit narrower

Reply:
let it be